core/domain/email_manager_test.py

```diff
@@ -1804,12 +1805,10 @@ def test_that_flag_exploration_emails_are_correct(self):
 
             # Make sure correct email models are stored.
             all_models = email_models.SentEmailModel.get_all().fetch()
-            all_models.sort(key=lambda x: x.recipient_id)
-            sent_email_model = all_models[0]
+            sent_email_model = python_utils.NEXT(
```

Comment:
Could you explain the reason behind this change (and the similar one below)?

Reply:
The sorting in the storage is different for some reason, it shouldn't have any other consequences because we don't assume anything about the ordering in the storage.